### PR TITLE
Sample for Spring Cloud CircuitBreaker Reactive;

### PR DESCRIPTION
**Pull request by marcingrzejszczak**
Related to spring-cloud/spring-cloud-sleuth#1910

### CODE REVIEW

**Comment by marcingrzejszczak** [comment]
cc @ryanjbaxter

File: circuitbreaker-reactive/src/main/resources/application.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+spring:
+  application:
+    name: circuitbreaker-reactive
+
+# For tests
+logging.level.org.springframework.cloud.sleuth: DEBUG
```
Comment (jonatan-ivanov) [inline circuitbreaker-reactive/src/main/resources/application.yaml:6]: new line at the end?

File: circuitbreaker-reactive/src/main/java/com/example/sleuthsamples/ReactiveCircuitBreakerApplication.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+			}), throwable -> {
+				log.info("<ACCEPTANCE_TEST> <TRACE:{}> Hello from producer", this.tracer.currentSpan().context().traceId());
+				return Mono.just("fallback");
+			});
+		}
+		finally {
+			span.end();
+		}
+	}
+}
```
Comment (jonatan-ivanov) [inline circuitbreaker-reactive/src/main/java/com/example/sleuthsamples/ReactiveCircuitBreakerApplication.java:63]: new line?

**Review by ryanjbaxter — APPROVED** [review]

**Comment by marcingrzejszczak** [comment]
Done via https://github.com/spring-cloud-samples/spring-cloud-sleuth-samples/commit/48c0f2054c8c2c5114211aba60d843ba45b03c8e

**Pull request closed without merging**